pcsx2/CDVD/IsoFS/IsoFS.cpp

```diff
@@ -49,8 +50,9 @@ IsoDirectory::IsoDirectory(SectorSource& r)
 	uint i = 16;
 
 	m_fstype = FStype_ISO9660;
-
-	while( !done )
+	
+	bool loadFromDisc = g_Conf->CdvdSource != CDVD_SourceType::NoDisc;
+	while( !done && loadFromDisc)
```

Comment:
I'm not talking about when the error message appears.

I'm saying this fix is in the wrong place. There is likely a better place to add code that prevents the error message from showing up.